vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-flow/src/main/resources/META-INF/resources/frontend/vaadin-spreadsheet/vaadin-spreadsheet.js

```diff
@@ -162,59 +170,33 @@ export class VaadinSpreadsheet extends LitElement {
     };
   }
 
-  static get styles() {
-    return css`
-      :host {
-        display: block;
-        height: 100%;
-        flex: 1 1 auto;
-        isolation: isolate;
-      }
+  constructor() {
+    super();
 
-      ::slotted(.v-spreadsheet) {
-        box-sizing: border-box;
-        min-height: 100px;
-      }
-    `
-  };
+    if (!overlayStyles.parentElement) {
+      // Append spreadsheet overlay styles to the document head
+      document.head.appendChild(overlayStyles);
+    }
+  }
 
   render() {
-    return html`
-      <slot></slot>
```

Comment:
Could also remove this JSDoc (seems like a leftover from some Lit starter):

https://github.com/vaadin/flow-components/blob/f4b726f965b9f3ca3478751e474e96ba67a7d5cb/vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-flow/src/main/resources/META-INF/resources/frontend/vaadin-spreadsheet/vaadin-spreadsheet.js#L15-L18

Reply:
Done. I'm planning to do a separate PR (to keep this PR as small as possible) to remove the leftover comments from the file and prettify it.